configs/openmrs_config/concepts/diagnosis.csv

```diff
@@ -1,4 +1,4 @@
-Uuid,Void/Retire,Same as mappings,Fully specified name:en,Fully specified name:km,Short name:en,Short name:km,Description:en,Description:fr,Data class,Data type,Answers,Members,_version:1,_order:1001,
```

Comment:
@kazlaw could you throw some light why the value is changing from `1001` to `1000`?

Reply:
Hey daud, put simply--> that `order` value sets priority when loading those csv files in `initialiser` meaning 1000 has higher priority than 1001. If you check `assessment.csv` it has the value of 1001, so when loading my diagnoses, the questions were not being loaded which are in `assessment.csv` because what `initialiser` does is it might randomly load those files yet a particular precedence is followed when loading Concepts and the mismatch my cause the issue I was facing 😀

Reply:
Could you set to something higher than `1001`? In case we need to have a file loaded in between in the future.
`2000` sounds better.

Reply:
Oh OK. We lowered it. Sorry I misunderstood.
After a deeper look, it seems we would need some rework of all files orders so they are not incremented by 1, but rather 100.
I'll approve this one but we need a follow up task.